srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -43,7 +44,7 @@ public Patient getPatient(final int dfn)
 	        final String[] fieldArray = results.get(0).split("\\^");
 	        final String patientName = fieldArray[0];
 	        final int patientAge = Integer.parseInt(fieldArray[1]);
-	        final String patientGender = fieldArray[2];
+	        final String patientGender = VistaFieldTranslation.TRANSLATION_MAP.get(fieldArray[2]);
```

Comment:
`get()` will return null if the map doesn't have a value. Should we handle that somehow?